tests/utils.go

```diff
@@ -529,10 +529,10 @@ func NewRandomVMWithDataVolumeAndUserDataInStorageClass(imageUrl, namespace, use
 //
 // Deprecated: Use libvmi directly
 func NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory(containerImage string, userData string) *v1.VirtualMachineInstance {
```

Comment:
The `containerImage` parameter is no longer used.
This could be handled in a follow-up.
WDYT?

Reply:
IMHO the follow up should be Remove the deprecated function :)

Reply:
We do plan to remove this function, I recall @orenc1 is on it.

Reply:
opened #11327